apps/src/templates/PuzzleRatingButtons.jsx

```diff
@@ -57,6 +69,10 @@ class PuzzleRatingButtons extends Component {
     };
   }
 
+  static propTypes = {
+    useLegacyStyles: React.PropTypes.bool,
```

Comment:
Do we need to support the old way of styling this component, or can we change the styles everywhere?